eo-runtime/src/main/java/org/eolang/Main.java

```diff
@@ -187,10 +187,11 @@ private static void run(final List<String> opts) throws Exception {
             phi.attr("Δ").put(new Data.Value<>(arg));
             app.attr(0).put(phi);
         }
-        if (!new Dataized(app).take(Boolean.class)) {
-            throw new IllegalStateException(
-                "Runtime dataization failure"
-            );
+        try {
+            final Boolean result = new Dataized(app).take(Boolean.class);
+            Main.LOGGER.info(String.format("The result of cast %s to bool is %s", app, result));
+        } catch (final ClassCastException exc) {
```

Comment:
@yasamprom this is a very bad practice, to catch this type of exception. Better use `take()` method (without param)